src/App.tsx

```diff
@@ -18,13 +18,19 @@ import User from './Components/User/User'
 import Ads from './Components/Ads/Ads'
 import Bookings from './Components/Bookings/Bookings'
 import Rooms from './Components/Rooms/Rooms'
+import Navbar from './Components/Navbar/Navbar'
+import { AuthContext } from './Context/AuthContext'
+import { useContext } from 'react'
 
 function App() {
+ 
+  let {userData,saveUserData,role}:any=useContext(AuthContext)
 const routes =createBrowserRouter([
   {
     path:"/",element:<AuthLayout/>,errorElement:<NotFound/>,
     children:[
       {index:true,element:<LandingPage/>},
+      {path:"nav",element:<Navbar/>},
```

Comment:
Navbar is part of the AppLayout but it is not a separate page with a distinct path.

Reply:
hwa 3shan fe 2 navbar f hawelt a5tre3 :'D